schemas/IDLs/org.gel.models.participant.avro/1.2.0/CommonParticipant.avdl

```diff
@@ -419,40 +417,54 @@ protocol CommonParticipant {
         union {null, string} clinicalIndicationVersionNumber;
     }
 
+    record Identifier{
```

Comment:
This name causes a collision with another record called `Identifier`.
Also, we already have something for this in the models called `Property` or alternatively a more complex one `StandardPhenotype`

Reply:
Yep Agree, I would do the changes tomorrow, first thing in the morning @priesgo